scw/config.go

```diff
@@ -16,20 +18,66 @@ const (
 	defaultConfigPermission = 0600
 )
 
+const configFileTemplate = `# Scaleway configuration file
+
+# You need an Access Key and a Secret Key to connect to Scaleway API.
+# Generate your token at the following address: https://console.scaleway.com/account/credentials
+
+# An Access Key is a Secret Key identifier.
+{{ if .AccessKey }}access_key: {{.AccessKey}}{{ else }}# access_key: SCW11111111111111111{{ end }}
+
+# The Secret Key is the value that can be used to authenticate against the API (the value used in X-Auth-Token HTTP-header).
+# The Secret Key MUST stay secret and not given to anyone or published online.
+{{ if .SecretKey }}secret_key: {{ .SecretKey }}{{ else }}# secret_key: 11111111-1111-1111-1111-111111111111{{ end }}
+
+# Your organization ID is the identifier of your account inside Scaleway infrastructure.
+{{ if .DefaultOrganizationID }}default_organization_id: {{ .DefaultOrganizationID }}{{ else }}# default_organization_id: 11111111-1111-1111-1111-111111111111{{ end }}
+
```

Comment:
Maybe we could explain that some products are regionalized and other are zoned here.

Reply:
We would need to change this lines everytime a new product is added in a new zone. I'm in favor of keeping it simple.